TMGToolbox/src/input_output/import_binary_matrix.py

```diff
@@ -69,7 +72,11 @@ class ImportBinaryMatrix(_m.Tool()):
     ImportFile = _m.Attribute(str)
     Scenario = _m.Attribute(_m.InstanceType) # common variable or parameter
     MatrixDescription = _m.Attribute(str)
-    
+
+    NewMatrixID = _m.Attribute(int)
+    NewMatrixName = _m.Attribute(str)
+    NewMatrixDescription = _m.Attribute(str)
+
     def __init__(self):
         #---Init internal variables
```

Comment:
I recommend initializing `self.NewMatrixName` and `self.NewMatrixDescription` to use empty strings, as otherwise you will get odd behaviour when turning `None` values into strings

Reply:
Changed as suggested.